### PR TITLE
fix: broken dynamic import of momentjs in timeline.vue

### PR DESCRIPTION
**Pull request by ivelin**
A recent dependency update broke dynamic imports in vue components.
The fix is to move the import to a static statement at the top of the vue component declaration.

### CODE REVIEW

**Comment by ivelin** [comment]
This is a fix for the regression spotted during peer review of #347 
@sniperadmin can you please review this PR.

File: src/views/Timeline.vue [hunk 1]
```diff
@@ -407,13 +408,13 @@ export default {
       return icon
     },
     friendlyTime (datetime) {
-      var moment = require('moment')
```
Comment (sniperadmin) [inline src/views/Timeline.vue:410]: aha! this (dynamic) guy was making the issue 👍 
good job, sir

**Comment by sniperadmin** [comment]
reviewed

File: src/views/Timeline.vue (hunk 1 above)
Comment (ivelin) [reply src/views/Timeline.vue:410]: Right. Thanks for the review. 
Will commit now and you can pull in the remote ID PR to continue the review there.

**Pull request merged**